x-pack/plugins/search_playground/public/components/view_query/view_query_flyout.tsx

```diff
@@ -68,14 +94,26 @@ export const ViewQueryFlyout: React.FC<ViewQueryFlyoutProps> = ({ onClose }) =>
       ...tempQueryFields,
       [index]: newFields,
     });
+    usageTracker.count(AnalyticsEvents.viewQueryFieldsUpdated, newFields.length);
   };
 
   const saveQuery = () => {
     queryFieldsOnChange(tempQueryFields);
     elasticsearchQueryChange(createQuery(tempQueryFields, fields));
     onClose();
+
+    const groupedQueryFields = groupFields(fields, tempQueryFields);
+
+    usageTracker.click(AnalyticsEvents.viewQuerySaved);
+    usageTracker.count(AnalyticsEvents.viewQuerySparseFields, groupedQueryFields.sparse);
```

Comment:
This isn't great metric in reality. We want a picture how many queries use:
- SPARSE
- DENSE
- BM25
- hybrid (SPARSE_BM25)
- hybrid (DENSE_BM25)
- Mix (SPARSE_DENSE_BM25)

basically if there is 1 or more fields from these grouping, add to string